src/language/from_relay/mod.rs

```diff
@@ -1808,4 +1813,306 @@ def @main(%x: Tensor[(3), float32]) -> Tensor[(3), float32] {
 (compute softmax (access (compute softmax (access (access-tensor x) 0)) 0))
 "#
     );
+
+    // TODO: enable this test once memoization goes in
+    //
+    // Creates a Relay-to-Glenside benchmark test over a model
+    // (currently the models we test are convolutional and only image-based)
+    // The test does the following:
+    //  1. Reads the $relay_str_path file and parses as relay module
+    //  2. Converts the relay module to glenside
+    //  3. Generates random input (uniform [0, 255]) and parameters (uniform [-1, 1])
+    //  4. Runs relay module through TVM and benchmarks running glenside expr through interpreter
+    //  5. Compare output of using TVM vs interpreter
+    // $test_name: the name of the created benchmark
+    // $relay_str_path: the path of the file containing the Relay code
+    macro_rules! benchmark_model {
+        ($test_name: ident, $relay_str_path:expr) => {
+            #[bench]
+            #[ignore = "this test causes stack overflow"]
```

Comment:
Is this still true?

Reply:
Ah, this is because you moved memoization out of this PR?

Reply:
yes, because this PR does not include memoization. (as we discussed memoization + ndarray optimizations should go into one PR)